geth-dev-node/genesis.json

```diff
@@ -0,0 +1,18 @@
+{
+  "config": {
+        "chainId": 10000,
+        "homesteadBlock": 0,
+        "eip155Block": 0,
+        "eip158Block": 0
+    },
+  "alloc"      : {},
+  "coinbase"   : "0x0000000000000000000000000000000000000000",
+  "difficulty" : "0x400",
+  "extraData"  : "",
+  "gasLimit"   : "9000000",
```

Comment:
Recommend setting this to 6.5M to be somewhat in line with the production network.

Reply:
Yeah, good idea.  I'll update this.